indico/modules/rb_new/controllers/backend/admin.py

```diff
@@ -47,12 +47,70 @@ def _process(self):
 
 
 class RHFeatures(RHRoomBookingAdminBase):
-    def _get_features(self):
+    def _process_args(self):
+        id_ = request.view_args.get('feature_id')
+        self.feature = RoomFeature.get_one(id_) if id_ is not None else None
+
+    def _dump_features(self):
         query = RoomFeature.query.order_by(RoomFeature.title)
-        return room_features_schema.dump(query).data
+        return room_feature_schema.dump(query, many=True).data
 
-    def _process(self):
-        return jsonify(self._get_features())
+    def _get_equipment_counts(self):
+        query = (db.session.query(equipment_features_table.c.feature_id, db.func.count())
+                 .group_by(equipment_features_table.c.feature_id))
+        return dict(query)
+
+    def _jsonify_one(self, equipment_type):
+        return jsonify(room_feature_schema.dump(equipment_type).data)
+
+    def _jsonify_many(self):
+        return jsonify(self._dump_features())
+
+    def _process_GET(self):
+        if self.feature:
+            return self._jsonify_one(self.feature)
+        else:
```

Comment:
What about removing `else` and saving one line?

Reply:
It's save 3 lines but I don't think it's more readable in this case

Reply:
I usually do that if the if block is just error handling or some other "exit early" case, but not if it's really two cases that are both "normal".